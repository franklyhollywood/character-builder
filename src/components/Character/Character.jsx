import React from 'react';

export default function Character({ top, middle, bottom }) {
	return (
		<React.Fragment>
			<br />
			<div
				aria-label='top'
				className='top'
				style={{
					backgroundImage: `url(./${top}-top.jpg)`,
					height: 150,
					width: 150,
					backgroundSize: 'cover',
					backgroundRepeat: 'no-repeat',
				}}
			></div>
			<div
				aria-label='middle'
				className='middle'
				style={{
					backgroundImage: `url(./${middle}-middle.jpg)`,
					height: 150,
					width: 150,
					backgroundSize: 'cover',
					backgroundRepeat: 'no-repeat',
				}}
			></div>
			<div
				aria-label='bottom'
				className='bottom'
				style={{
					backgroundImage: `url(./${bottom}-bottom.jpg)`,
					height: 150,
					width: 150,
					backgroundSize: 'cover',
					backgroundRepeat: 'no-repeat',
				}}
			></div>
		</React.Fragment>
	);
}
