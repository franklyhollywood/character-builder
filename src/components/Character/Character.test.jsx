import { screen, render } from '@testing-library/react';
import Character from './Character';

it('should test da page', () => {
	render(<Character top='fish' middle='torso' bottom='legs1' />);
	const top = screen.getByLabelText('top');
	expect(top).toMatchSnapshot();
});
