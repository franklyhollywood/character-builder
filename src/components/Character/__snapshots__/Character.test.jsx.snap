// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test da page 1`] = `
<div
  aria-label="top"
  class="top"
  style="background-image: url(./fish-top.jpg); height: 150px; width: 150px; background-size: cover; background-repeat: no-repeat;"
/>
`;
